core/templates/services/assets-backend-api.service.ts

```diff
@@ -91,27 +91,7 @@ angular.module('oppia').factory('AssetsBackendApiService', [
       }).then(function(response) {
         var assetBlob = null;
         var data = response.data;
-        try {
-          assetBlob = new Blob([data], {type: data.type});
-        } catch (exception) {
-          window.BlobBuilder = window.BlobBuilder ||
```

Comment:
Why are we deleting these? [This helps us to have this functionality in cross-browser/devices.]

Reply:
You can use `git blame` to check when and why this was introduced*

Reply:
According to @vojtechjelinek all the browsers have this `new Blob()` now.

Reply:
all the new updated/browsers, right? (Have you check why this was added in the first place? Also, might worth checking server log errors for the errors dict reported through this function)

Reply:
https://github.com/oppia/oppia/pull/9278#discussion_r422494939

@kevintab95 Can you please check the server logs for the message that we are deleting?

Reply:
Per my [comment](https://github.com/oppia/oppia/pull/8522#issuecomment-606160204) on a different PR, these were added to debug #6162 and that hasn't occured in a while. It seems to be fixed by #7784 where we added a second argument to the Block constructor. So as long as we use Blob with the second argument, it should be fine.

Reply:
Thanks for the info, @kevintab95!